scripts/core/context.yml

```diff
@@ -106,16 +106,19 @@ params:
     - type: $x_context_info_t
       name: ContextInfoType
       desc: "[in] type of the info to retrieve"
-    - type: "size_t*"
-      name: pSize
+    - type: "size_t"
+      name: Size
       desc: |
-            [in,out] pointer to the number of bytes needed to return info queried.
-            the call shall update it with the real number of bytes needed to return the info
+            [in] the number of bytes of memory pointed to by pContextInfo.
     - type: "void*"
       name: pContextInfo
       desc: |
             [out][optional] array of bytes holding the info.
-            if *pSize is not equal to the real number of bytes needed to return the info then the $X_RESULT_ERROR_INVALID_SIZE error is returned and pContextInfo is not used.
+            if Size is not equal to the real number of bytes needed to return the info then the $X_RESULT_ERROR_INVALID_SIZE error is returned and pContextInfo is not used.
```

Comment:
Is this to restrictive? Couldn't size be greater than the value returned via `pSizeRet`? In OpenCL the wording is something like:
```
... returns `CL_INVALID_VALUE` if size in bytes specified by `param_value_size` is < size of return type.
```
The entry point is going to write `*pSizeRet` bytes into `pContextInfo` so as long as the memory at `pContextInfo` is at least `*pSizeRet` bytes we should be good right?

Reply:
👍